models/custom-application/src/custom-application/generator.ts

```diff
@@ -0,0 +1,47 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import * as CustomApplicationDeploymentPreview from '../custom-application-deployment-preview';
+import * as CustomApplicationMenuLink from '../custom-application-menu-link';
+import * as CustomApplicationPermission from '../custom-application-permission/custom-application-permission';
+import * as CustomApplicationSubmenuLink from '../custom-application-submenu-link';
+
+import type { TCustomApplication } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+function slugify(text: string) {
+  return text
+    .toString()
+    .toLowerCase()
+    .replace(/\s+/g, '-') // Replace spaces with -
+    .replace(/[^\w-]+/g, '') // Remove all non-word chars
+    .replace(/--+/g, '-') // Replace multiple - with single -
+    .replace(/^-+/, '') // Trim - from start of text
+    .replace(/-+$/, ''); // Trim - from end of text
+}
```

Comment:
Nit: move it to a util?

Reply:
updated [834559c](https://github.com/commercetools/test-data/pull/542/commits/834559cc35ba3e721144156abee033e728237e7f)